src/components/WwwFrame/Menus/TheSettingsTertiaryMenu.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<tertiary-menu>
+		<ul>
+			<li>
+				<router-link
```

Comment:
Not sure if these are exactly the links we want

Reply:
With the exception of Security settings, which may need a feature flag, the other links show in the legacy settings sub nav too.

We could add Saved searches to fully match the two apps, though this will require a bit more setup.